Aboutme.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var answer = prompt('Do you like scary stories?');
+if (answer.toLowerCase() === 'yes' || answer.toLowerCase() === 'y') {
+  alert('BOO! (it\'s a short story)');
+} else if (answer.toLowerCase() === 'no' || answer.toLowerCase() === 'n') {
+  alert('It\'s your loss.');
+} else {
+  alert('No need to get scared and start typing out control');
+}
+
+var answer = prompt('Are you ready to party?');
+if (answer.toLowerCase() === 'yes' || answer.toLowerCase() === 'y') {
+  alert('OKAY! You go buy the Beer');
+}
+else if (answer.toLowerCase() === 'no' || answer.toLowerCase() === 'n') {
+  alert('OK');
+} else {
+  alert('I\'ll take that as a no.');
+}
+
+var answer = prompt('Do you like Superman?');
+if (answer.toLowerCase() === 'yes' || answer.toLowerCase() === 'y') {
+  alert('Superman is cool.');
+}else if (answer.toLowerCase() === 'no' || answer.toLowerCase() === 'n') {
```

Comment:
to stay consistent with the rest of your code, put a space between the `{` and the `else if`